packages/client/tests/functional/composites-object/findMany.ts

```diff
@@ -104,7 +85,9 @@ describeIf(!process.env.TEST_SKIP_MONGODB)('findMany > required', () => {
       },
     })
 
-    expect(comment).toMatchInlineSnapshot(`
+    expect(comment).toMatchInlineSnapshot(
```

Comment:
WDYT about using `expect(result).toMatchObject({})` over snapshots for the tests in this PR? This way we can use truly random data.

Reply:
Not really sure about it in this case, to be honest. It will be mostly the same snapshot, but written and updated manually instead.